### PR TITLE
Allow setting ENFORCE_GLOBAL_UNIQUE in env

### PR DESCRIPTION
**Pull request by Danmark**

### CODE REVIEW

File: docker/configuration.docker.py [hunk 1]
```diff
@@ -77,3 +77,9 @@
 SHORT_TIME_FORMAT = os.environ.get('SHORT_TIME_FORMAT', 'H:i:s')
 DATETIME_FORMAT = os.environ.get('DATETIME_FORMAT', 'N j, Y g:i a')
 SHORT_DATETIME_FORMAT = os.environ.get('SHORT_DATETIME_FORMAT', 'Y-m-d H:i')
+
+# Enforcement of unique IP space can be toggled on a per-VRF basis.
+# To enforce unique IP space within the global table (all prefixes and IP addresses not assigned to a VRF),
+# set ENFORCE_GLOBAL_UNIQUE to True.
+ENFORCE_GLOBAL_UNIQUE = ENFORCE_GLOBAL_UNIQUE = os.environ.get('ENFORCE_GLOBAL_UNIQUE', '')
```
Comment (cimnine) [inline docker/configuration.docker.py]: What's the purpose of this part: `ENFORCE_GLOBAL_UNIQUE = ENFORCE_GLOBAL_UNIQUE`?

Comment (Danmark) [reply docker/configuration.docker.py]: Yeah. That's just wrong.

File: docker/configuration.docker.py [hunk 2]
```diff
@@ -77,3 +77,9 @@
 SHORT_TIME_FORMAT = os.environ.get('SHORT_TIME_FORMAT', 'H:i:s')
 DATETIME_FORMAT = os.environ.get('DATETIME_FORMAT', 'N j, Y g:i a')
 SHORT_DATETIME_FORMAT = os.environ.get('SHORT_DATETIME_FORMAT', 'Y-m-d H:i')
+
+# Enforcement of unique IP space can be toggled on a per-VRF basis.
+# To enforce unique IP space within the global table (all prefixes and IP addresses not assigned to a VRF),
+# set ENFORCE_GLOBAL_UNIQUE to True.
+ENFORCE_GLOBAL_UNIQUE = os.environ.get('ENFORCE_GLOBAL_UNIQUE', '')
```
Comment (cimnine) [inline docker/configuration.docker.py]: If this is a boolean, shouldn't the default be `false` instead of an empty string?

Comment (Danmark) [reply docker/configuration.docker.py]: My intention was to use the default set here if it was undefined in the env https://github.com/digitalocean/netbox/blob/252be84bf07a27868c9e5548558834b2fd62e3d8/netbox/netbox/settings.py#L44

Hence there should be no default.
I fix!

**Comment by cimnine** [comment]
As soon as [the build](https://travis-ci.org/ninech/netbox-docker/builds/310141932) ran through, the option will be available in the Docker image pushed to [Docker Hub](https://hub.docker.com/r/ninech/netbox/).

**Pull request merged**